lib/ex_statsd.ex

```diff
@@ -67,7 +67,7 @@ defmodule ExStatsD do
   It returns the amount given as its first argument, making it suitable
   for pipelining.
   """
-  def counter(amount, metric, options \\ [sample_rate: 1, tags: []]) do
+  def counter(amount, metric, options \\ default_options) do
```

Comment:
Minor note, can you make this `default_options()`? I think it makes it clearer that this is a function call.